### ISSUE TITLE
Multi Column drop down

### ISSUE THREAD

**Issue by paddyji** [LeaVerou/awesomplete#17122]
I would like a multi-column drop down. because in case of common names like John Pintos in a Customer database, then the second column becomes an identifier for the specific 'John Pinto' 

Rgds.
Pradeep Chavan

**Comment by matthewblott** [comment]
I was looking for the same. I wanted a description to go with a stock code lookup field and have compromised with ```stockCode + ' - ' description``` as the label field. Works for now but I can see situations where it'd be good to have more than one column. I had a quick look but was a bit lost. Any chance this can be added as an enhancement? Thanks in advance :-)

**Comment by LeaVerou** [comment]
Have you tried doing this with CSS, via either the `columns` property or grid layout? Depending on what you're trying to do, it may work.
I can provide more help if you tell me what exactly you're trying to accomplish (screenshots/demos are better than words).

**Comment by matthewblott** [comment]
Okay, this is what I have:

![s 2019-10-31 at 11 08 07](https://user-images.githubusercontent.com/1231417/67942025-c8127b80-fbce-11e9-8b8d-e9ff23b7800f.jpg)

It would be nice to have the stock code and description in separate columns. I wasn't aware the ```column``` tag could be used with the ```option``` tag like so:

```

<select>
    <option value="1">
        <column>001</column>
        <column>Widget 1</column>
    </option>
    <option value="2">
        <column>002</column>
        <column>Widget 2</column>
    </option>
</select>

```

According to [this](https://stackoverflow.com/questions/17162219/html-multiple-column-drop-down-with-adjusted-columns) StackOverflow answer the above isn't valid HTML. It's not a biggie but any help appreciated. Thanks :-)

**Comment by paddyji** [comment]
Telerik controls have the multi-column search in C#, but I have not found
it in any web components, maybe a little R&D on CSS Grid might do the trick.
I am trying.
Rgds,
Pradeep Chavan.

**Comment by matthewblott** [comment]
@paddyji Telerik controls are also pretty damn expensive as I remember!

**Comment by paddyji** [comment]
Yes, Hopefully now with smart CSS3, they will come to their senses.

Rgds,
Pradeep Chavan.
